onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4736,12 +4736,21 @@ def aten_to_sparse_csr(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
+@torch_op("aten::topk", trace_only=True)
```

Comment:
Can this be compiled?

Reply:
Done. Thx for tip on `input_wrangler`. Do we have an issue tracking the attr-kwargs-onnxscript limitation?

Reply:
Just added https://github.com/microsoft/onnx-script/issues/305